src/app/regular-giving/regular-giving.component.html

```diff
@@ -59,6 +59,74 @@
                 >Continue</button>
               </div>
 
+            </mat-step>
+            <mat-step
+              formGroupName="amounts"
+              label="Your details"
+              class="c-your-donation">
+
+              <p>
+                The details below are set in <a href="/my-account">your account</a> and will be used for these
+                donations. They are not editable.
+                If these details are wrong please <a href="#" (click)="logout()">log out of this account and </a> create a new
+                account if necassary, or contact Big Give.
+              </p>
+
+              <biggive-text-input>
+                <label
+                  class="label-with-limited-space"
+                  slot="label"
+                  for="donationAmount"
+                >
+                  First Name
+                </label>
+                <input
+                  readonly="readonly"
+                  value="{{donor.first_name}}"
+                  slot="input"
+                />
+              </biggive-text-input>
+
+              <biggive-text-input>
+                <label
+                  class="label-with-limited-space"
+                  slot="label"
+                  for="donationAmount"
+                >
+                  Last Name
+                </label>
+                <input
+                  readonly="readonly"
+                  value="{{donor.last_name}}"
+                  slot="input"
+                />
+              </biggive-text-input>
+
+              <biggive-text-input>
+                <label
+                  class="label-with-limited-space"
+                  slot="label"
+                  for="donationAmount"
+                >
+                  Email Address
+                </label>
+                <input
+                  readonly="readonly"
+                  value="{{donor.email_address}}"
+                  slot="input"
+                />
+              </biggive-text-input>
```

Comment:
not 100% sure if we should use read only input elements here - it's not visually obvious that they are readonly. We could just make a table or something to display these details, which might be clearer but need a bit more styling work.

Reply:
I think that readonly with no styling changes will likely confuse and frustrate.

I think most browsers give `disabled` fields more obvious default styling – is there a reason to avoid that attr here?

In any case, I think we would do well to pick some simple styling of our own for whichever state we choose – at a minimum, a slightly darker grey background and a 'not allowed' cursor for desktop.

Reply:
@NoelLH No particular reason not to have more obvious styling for disabled no - I didn't turn any styling off. Can add some disabled style, but do you think we even want to use input elements here? I'm leaning towards actually probably not. I can change it to use a list or table or something like that with no suggestion of interactivity.

Reply:
@NoelLH do you have a view on if input elements are useful here?

Reply:
Changing the background is a bit complicated since it needs to go through the component library. I think I'll try just not using inputs.

Reply:
I could simplify it to this. Although we store first & last name separately we display together on the account page, probably can do the same here. Maybe this doesn't need to actually have its own step though - we could just show these as part of the confirmation step.

![image](https://github.com/user-attachments/assets/8ed60149-96cf-4fee-83e4-d49adcf80ba4)

Reply:
On my local I just moved these down into the confirm step like so: 
![image](https://github.com/user-attachments/assets/bf95eb1b-f899-46e8-884c-f59017d4dd2b)

On my sketch I also had the editable fields billing country and billing postcode in this step, but I think it actually makes more sense to do them in the same step as the payment card selection, so I've pushed that change to this branch as 9795a5c2.